README.md

```diff
@@ -1,2 +1,157 @@
+    ```
+
+
+### Mac OS
+
+**In order to install required software on your mac, you will need GCC and Homebrew. If you don't have them installed already, refer to the steps below:**
+
+1. GCC can be obtained by downloading XCode, Command Line Tools or OSX-GCC-Installer package. If you already have XCode installed, **DO NOT** install OSX-GCC-Installer. If you perform a fresh install of XCode, you will also need to add the commandline tools by running `xcode-select --install` on the terminal.
+2. To install Homebrew run the following command in Terminal 
+        `ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"`
+    The script will explain what changes it will make and prompt you before the installation begins. Once you’ve installed Homebrew, insert the Homebrew directory at the top of your PATH environment variable. You can do this by adding the following line at the bottom of your ~/.profile file
+    ```
+    export PATH=/usr/local/bin:/usr/local/sbin:$PATH
+    ```   
+**This project uses Python in version 3.7.0. If you're already using the same or higher version, you can skip the following step.**
+
+3. Run Homebrew installation of Python in Terminal with `brew install python`.
+    You can verify the installation by running `python3 --version`.
+
+**Skip this step if PIP is already installed.**
+
+4. To install PIP run `sudo easy_install pip` in Terminal.
+
+5. If you don't have virtualenv installed, run installation with PIP by typing `pip install virtualenv` in Terminal
+
+6. In Terminal, navigate to directory you wish to install virtual environment to and type `virtualenv <name for your venv>`.
+You can now activate your virtual environment by running `source <your venv location>/bin/activate`. To deactivate it simply type `deactivate` in Terminal.
+
+7. To install PostgreSQL run 'brew install postgresql'.
+    If this is your first install, create a database with: 'initdb /usr/local/var/postgres'.
+
+8.  Edit pg_hba.conf file located in */usr/local/var/postgres/* adding the following line:
+    ```
+    local   all             postgres                                trust 
+    ```
+
+9. Start PostgreSQL service with `brew services start postgresql`. You can also use commands ***restart*** and ***stop*** to control the service. If for some reason you have problem with managing service, you can also control it manually with `pg_ctl -D /usr/local/var/postgres <command>`.
+
+10. Create database for application by running the following command in Terminal: `createdb -U postgres time_monkey`.
+
+11. Navigate with Terminal to where you want the repository to be cloned and run `git clone https://github.com/Code-Poets/time-monkey.git`
+
+12. Use requirements.lock to install required pacakges to your virtual environment by running `pip install -r requirements.lock`
+
+13. Add file to */time_monkey/settings* directory named *local_settings.py* and fill with following content:
```

Comment:
You have to write it one default version for all systems because as far as I can see everywhere is same instruction for `local_settings.py`

Reply:
Done